protractor/e2e/integration/ng2.spec.ts

```diff
@@ -58,5 +58,13 @@ describe('Main page test suite', () => {
         await secondPage.infoSubmitButton.click();
     });
 
+    it('Check blog page',async () => {
+        await secondPage.blogButton.click();
+        expect(await secondPage.getCurrentUrl()).toContain(secondPage.blogLink);
+        expect(await secondPage.postsList.count()).toEqual(20);
```

Comment:
And user see 20 articles with header, description and link - this is task
You check only headers in this test. So, need to add more checks, like:
const expectedArticlesLength = 20;
expect(await $$('article').count()).toBe(expectedArticlesLength);
        expect(await $$('article header').count()).toBe(expectedArticlesLength);
        expect(await $$('article .post-entry').count()).toBe(expectedArticlesLength);
        expect(await $$('article a[class*="btn-action"]').count()).toBe(expectedArticlesLength);